pkg/kubernetes/app.js

```diff
@@ -30,6 +30,16 @@ define([
             calculated.containers = "0";
 
             /*
+             * Calculate total number of containers = replicas * containersArray.length
+             */
+            client.select(service.spec.selector || { },
+                          service.metadata.namespace, "ReplicationController").forEach(function(rc) {
```

Comment:
This is an interesting approach. But I don't this actually works.

In particular, both services and reciplicationcontrollers select pods. But services do not select replicationcontrololers, or vice versa.